### PR TITLE
feat: Introduce AcceptListStargateQuerier

### PR DESCRIPTION
**Pull request by dudong2**
## Description
This PR is based on https://github.com/CosmWasm/wasmd/pull/1069

```go
accepted := wasmkeeper.AcceptedStargateQueries{
	"/lbm.collection.v1.Query/Contract": &collection.QueryContractResponse{},
}
querierOpts := wasmkeeper.WithQueryPlugins(
	&wasmkeeper.QueryPlugins{
		Stargate: wasmkeeper.AcceptListStargateQuerier(accepted, app.GRPCQueryRouter(), appCodec),
	})
wasmOpts = append(wasmOpts, querierOpts)

// The last arguments can contain custom message handlers, and custom query handlers,
// if we want to allow any custom callbacks
availableCapabilities := "iterator,staking,stargate,cosmwasm_1_1"
app.WasmKeeper = wasmpluskeeper.NewKeeper(
	appCodec,
	keys[wasmplustypes.StoreKey],
	app.GetSubspace(wasmplustypes.ModuleName),
	app.AccountKeeper,
	app.BankKeeper,
	app.StakingKeeper,
	app.DistrKeeper,
	app.IBCKeeper.ChannelKeeper,
	&app.IBCKeeper.PortKeeper,
	scopedWasmKeeper,
	app.TransferKeeper,
	app.MsgServiceRouter(),
	app.GRPCQueryRouter(),
	wasmDir,
	wasmConfig,
	availableCapabilities,
	wasmOpts...,
)
```

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/wasm/keeper/reflect_test.go [hunk 1]
```diff
@@ -406,7 +406,7 @@ func TestReflectInvalidStargateQuery(t *testing.T) {
 	// make a query on the chain, should be blacklisted
```
Comment (da1suk8) [inline x/wasm/keeper/reflect_test.go:406]: `should not be whitelisted`?

File: x/wasm/keeper/reflect_test.go [hunk 2]
```diff
@@ -423,7 +423,7 @@ func TestReflectInvalidStargateQuery(t *testing.T) {
...
```
Comment (da1suk8) [inline x/wasm/keeper/reflect_test.go:423]: `should not be whitelisted`?

File: x/wasm/keeper/reflect_test.go (hunk 1 above)
Comment (dudong2) [reply x/wasm/keeper/reflect_test.go:406]: `cosmwasm/wasmd` has the exactly same code. This PR simply focuses on cherry-picking [CosmWasm#1069](https://github.com/CosmWasm/wasmd/pull/1069). I also think it's better to keep the code in the same state as `cosmwasm/wasmd` unless it's a necessary fix. So I don't think a fix is necessary.

File: x/wasm/keeper/reflect_test.go (hunk 2 above)
Comment (dudong2) [reply x/wasm/keeper/reflect_test.go:423]: ditto

File: x/wasm/keeper/reflect_test.go (hunk 1 above)
Comment (da1suk8) [reply x/wasm/keeper/reflect_test.go:406]: Understood thanks.

**Review by da1suk8 — APPROVED** [review]

**Review by loloicci — CHANGES_REQUESTED** [review]
I think this PR should be merged into main, not to release/v0.1.x

**Review by da1suk8 — APPROVED** [review]

**Review by loloicci — APPROVED** [review]

**Pull request merged**